Shell/LineEditor.cpp

```diff
@@ -354,15 +388,43 @@ String LineEditor::get_line(const String& prompt)
                 }
 
                 String token = is_empty_token ? String() : String(&m_buffer[token_start], m_cursor - (size_t)token_start);
+                Vector<String> suggestions;
 
                 if (is_first_token)
-                    tab_complete_first_token(token);
+                    suggestions = tab_complete_first_token(token);
                 else
-                    tab_complete_other_token(token);
+                    suggestions = tab_complete_other_token(token);
+
+                if (m_times_tab_pressed > 1 && !suggestions.is_empty()) {
+                    size_t longest_name = 0;
```

Comment:
This is not the longest name, this is its length, so maybe `longest_name_length` or `length_of_longest_name` or `max_name_length`. And besides these aren't even "names", they're "suggestions", aren't they?

Reply:
In the context of this loop it's the longest name, though YMMV with how it's interpreted. `longest_name_length` is a but more concrete so I'll change it to that. 

> And besides these aren't even "names", they're "suggestions", aren't they?
True, but they're "names" of files. Or at least that's my reasoning.

Reply:
> True, but they're "names" of files. Or at least that's my reasoning.

That won't be the case when we add more completion sources :D